File: app/src/main/java/com/iceteaviet/fastfoodfinder/data/remote/routing/MapsRoutingApiHelper.java
package com.iceteaviet.fastfoodfinder.data.remote.routing;

import com.iceteaviet.fastfoodfinder.data.remote.routing.model.MapsDirection;
import com.iceteaviet.fastfoodfinder.data.remote.store.model.Store;

import java.util.Map;

import io.reactivex.Single;

/**
 * Created by tom on 7/18/18.
 */
public interface MapsRoutingApiHelper {
    Single<MapsDirection> getMapsDirection(Map<String, String> queries, final Store store);
}

File: app/src/main/kotlin/com/iceteaviet/fastfoodfinder/data/remote/routing/MapsRoutingApiHelper.kt
package com.iceteaviet.fastfoodfinder.data.remote.routing

import com.iceteaviet.fastfoodfinder.data.remote.routing.model.MapsDirection
import com.iceteaviet.fastfoodfinder.data.remote.store.model.Store

import io.reactivex.Single

/**
 * Created by tom on 7/18/18.
 */
interface MapsRoutingApiHelper {
    fun getMapsDirection(queries: Map<String, String>, store: Store): Single<MapsDirection>
}
